azure-pipelines.yml

```diff
@@ -38,84 +38,82 @@ strategy:
     #      | HEAD         |  >= 8.10.1  | (since 2021/03/10, https://gitlab.haskell.org/ghc/ghc/-/commit/0a709dd9876e40c19c934692415c437ac434318c)
     #      +--------------+-------------+
 
+    # +---------+-----------------+------------+
+    # | OS      | ghc-lib flavour | GHC        |
+    # +=========+=================+============+
+    # | linux   | ghc-master      | ghc-9.2.1  |
+    # | macOS   | ghc-master      | ghc-9.2.1  |
+    # +---------+-----------------+------------+
+    linux-ghc-master-9.2.1:
```

Comment:
I’m a bit worried that we just keep extending the number of CI jobs. I think we should start thinking about which ones we can drop reasonably. Maybe 8.8 can go (excluding the daml fork), maybe we can limit cross-platform tests to only the most recent one.

Nothing for this PR but I think we have to address this soonish.